src/intercept_desc.c

```diff
@@ -0,0 +1,632 @@
+/*
+ * Copyright 2016-2017, Intel Corporation
```

Comment:
Why does this copyright include 2016? It isn't clear to me if the copyright is from the date of inception of code or the date of some sort of publication. I am just curious.